README.md

```diff
@@ -150,9 +154,27 @@ $ npm run package:arm64             # prepare `build` directory for arm64
 
 $ npm run prepare:all               # prepare phonegap platform files
 
-$ npm run start:emulator            # run on emulator, alternatively use start:device
+if you have iOS enabled, since Xcode10 we have to pass buildFlag
+$ npm run build -- --buildFlag='-UseModernBuildSystem=0'

```

Comment:
Is that true? I was a able to build just fine without it; e.g. opening the project in XCode and building.
By default XCode uses the new build system.

Reply:
Removed